### PR TITLE
chore(ST-67): created user student and teacher table.

### PR DESCRIPTION
**Pull request by ghost**
## Description of Change
- Created User Student and Teacher's table.
- Delete Unnecessary files like roles, userProfile.
- Added enums for student and user

## Associated Ticket Link(s)

- https://trello.com/c/DlhtaTgt

## Screenshots / Screen Recordings
N/A

### CODE REVIEW

File: src/common/enums/student.enums.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export enum EducationLevel {
```
Comment (ragib-hossain-sazim) [inline src/common/enums/student.enums.ts]: `EducationLevel` => `EEducationLevel`

File: src/common/enums/student.enums.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    SCHOOL = 'school',
+    COLLEGE = 'college',
+    UNIVERSITY = 'university',
+}
+
+export enum Medium {
```
Comment (ragib-hossain-sazim) [inline src/common/enums/student.enums.ts]: `Medium` => `EMedium`

File: src/common/enums/userType.enums.ts
Comment (ragib-hossain-sazim) [inline src/common/enums/userType.enums.ts:1]: `userType.enums.ts` => `users.enums.ts`

File: src/common/entities/student.entity.ts
Comment (ragib-hossain-sazim) [inline src/common/entities/student.entity.ts:1]: file names should be plural `student.entity.ts` => `students.entity.ts`

**Comment by ragib-hossain-sazim** [comment]
commit message should be `feat` not `chore` in this case.

File: src/common/entities/students.entity.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  id!: number;
+
+  @Enum(() => EEducationLevel)
+  @Property({ fieldName: 'education_level' })
+  educationLevel!: EEducationLevel;
+
+  @Enum(() => EMedium)
+  medium?: EMedium;
+
+  @Property()
```
Comment (ishmam-iqbal-sazim) [inline src/common/entities/students.entity.ts]: fieldName should be explicitly provided for all entity attributes

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending comments.

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**